app/assets/javascripts/lara-typescript.js

```diff
@@ -29314,7 +29333,9 @@ exports.ManagedInteractiveAuthoring = function (props) {
                         React.createElement("legend", null, "Url Fragment"),
                         React.createElement("textarea", { id: formField("url_fragment").id, name: formField("url_fragment").name, defaultValue: url_fragment, onBlur: handleUrlFragmentBlur })),
                     libraryInteractive.authoring_guidance
-                        ? React.createElement("div", { dangerouslySetInnerHTML: { __html: libraryInteractive.authoring_guidance } })
+                        ? React.createElement("fieldset", null,
+                            React.createElement("legend", null, "Authoring Guidance"),
+                            React.createElement("div", { dangerouslySetInnerHTML: { __html: libraryInteractive.authoring_guidance } }))
```

Comment:
Not necessarily part of this PR since it's not a change, but elsewhere we've been using a [renderHTML()](https://github.com/concord-consortium/question-interactives/blob/master/src/shared/utilities/render-html.ts) utility function to replace use of `dangerouslySetInnerHTML`).